approval.go

```diff
@@ -97,6 +98,27 @@ Respond %s to continue workflow or %s to cancel.`,
 	return nil
 }
 
+func (a *approvalEnvironment) saveOutput(ctx context.Context) (bool, error) {
```

Comment:
I think this should be a shared function for allowing for future outputs to be added at the appropriate time 

```suggestion
func (a *approvalEnvironment) SetActionOutputs(outputs map[string]string, ctx context.Context) (bool, error) {
```

Reply:
also, is `ctx` used?

Reply:
Thanks for the suggestion. Just pushed it.